backend/de.metas.material/dispo-service/src/main/java/de/metas/material/dispo/service/candidatechange/StockCandidateService.java

```diff
@@ -282,14 +291,14 @@ private MaterialDescriptorQueryBuilder createMaterialDescriptorQueryBuilder(
 				.builder()
 				.customerIdOperator(CustomerIdOperator.GIVEN_ID_OR_NULL); // want the latest, only excluding records that have a *different* customerId
 
-		if (materialDescriptor.getCustomerId() != null)
+		if (materialDescriptor.getCustomerId() != null && materialDescriptor.isReservedForCustomer())
 		{
 			// do include the bpartner in the query, because e.g. an increase for a given bpartner does a raised ATP just for that partner, and not for everyone
```

Comment:
```suggestion
			// do include the bpartner in the query, because e.g. an increase for a given bpartner causes a raised ATP just for that partner, but not for everyone
```